src/Commands/SettingsPublisher.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace CodeIgniter\Settings\Commands;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\CLI;
+use CodeIgniter\Publisher\Publisher;
+use Throwable;
+
+/**
+ * Publish Settings config file into the current application.
+ */
+class SettingsPublisher extends BaseCommand
+{
+    /**
+     * The group the command is lumped under
+     * when listing commands.
+     *
+     * @var string
+     */
+    protected $group = 'Settings';
+
+    /**
+     * The Command's name
+     *
+     * @var string
+     */
+    protected $name = 'settings:publish';
+
+    /**
+     * the Command's usage
+     *
+     * @var string
+     */
+    protected $usage = <<<'EOL'
+        settings:publish [options]
+
+          Examples:
+            settings:publish -f
+        EOL;
+
+    /**
+     * the Command's short description
+     *
+     * @var string
+     */
+    protected $description = 'Publish Settings config file into the current application.';
+
+    /**
+     * The Command's options
+     *
+     * @var array<string, string>
+     */
+    protected $options = [
+        '-f' => 'Set to enable overwrites.',
+    ];
+
+    /**
+     * @var bool `true` to enable overwrites file
+     */
+    private bool $overwrites = false;
+
+    public function run(array $params): void
+    {
+        if (array_key_exists('f', $params)) {
+            $this->overwrites = true;
+        }
+
+        // Use the Autoloader to figure out the module path
+        $source    = service('autoloader')->getNamespace('CodeIgniter\\Settings')[0];
+        $publisher = new Publisher($source, APPPATH);
+
+        try {
+            $publisher->addPath('Config/Settings.php')
+                ->merge($this->overwrites);
```

Comment:
Can you align `->addPath` to the `->merge` below?
```suggestion
            $publisher
                ->addPath('Config/Settings.php')
                ->merge($this->overwrites);
```